static/src/images/global/market-up.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="15" height="11" viewBox="0 0 15 11"><path fill="#298422" d="M0 9.97L1.03 11H14l1-1L8 0h-.958L0 9.984z"/></svg>
```

Comment:
@robertberry, terribly sorry - just came in... I am not sure if you change the default svo config the same way if the svgo is run as part of grunt, but for the standalone node-based one, you have to reference a config yml file as per https://github.com/svg/svgo/blob/master/docs/how-it-works/en.md#1-config where you override default values like e.g. here: https://github.com/fontello/svg-font-create/blob/master/svgo.yml (``floatPrecision`` arg of ``convertPathData`` plugin is important here and should be set to 1; 0 would give even smaller files, but 1 is already slightly dangerous - to use 1 the input SVGs would have to be 10x size to not result in garbled paths). Let me know if I can help more...